extensions/wikidata/module/langs/translation-en.json

```diff
@@ -265,12 +265,12 @@
     "warnings-messages/new-media-without-wikitext/title": "New media without wikitext",
     "warnings-messages/new-media-without-wikitext/body": "Uploading new media files requires supplying accompanying wikitext. No wikitext was provided for {example_entity}, for instance.",
     "wikibase-statement-settings/dialog-header": "Statement settings",
-    "wikibase-statement-settings/mode-label": "Editing mode: ",
-    "wikibase-statement-settings/strategy-label": "Matching strategy: ",
-    "wikibase-statement-settings/strategy-options": "Strategy options: ",
+    "wikibase-statement-settings/mode-label": "Editing mode ",
+    "wikibase-statement-settings/strategy-label": "Matching strategy ",
+    "wikibase-statement-settings/strategy-options": "Strategy options ",
     "wikibase-statement-settings/no-strategy-options": "No options for this matching strategy.",
     "wikibase-statement-settings/lax-value-matching": "Lax value matching (ignore insignificant differences between values, such as trailing whitespace or rounding differences)",
-    "wikibase-statement-settings/discriminating-qualifiers-label": "Discriminating qualifiers: ",
+    "wikibase-statement-settings/discriminating-qualifiers-label": "Discriminating qualifiers ",
```

Comment:
Is it not a problem to have trailing whitespace in those strings?

Reply:
It is, except for that one (Discriminating Qualifiers) but those labels shouldn't have whitespace anyways so I removed it from them and added styling to compensate for that one. The screenshot is updated as well.